src/test/regress/expected/multi_mx_node_metadata.out

```diff
@@ -9,6 +9,7 @@ SET citus.replication_model TO streaming;
 SET citus.shard_count TO 8;
 SET citus.shard_replication_factor TO 1;
 SET citus.replicate_reference_tables_on_activate TO off;
+\set VERBOSITY terse
```

Comment:
@onderkalaci this line, along with the removed `DETAIL:` lines caused a merge conflict on enterprise. I will resolve that.

That being said, should we take a look at enterprise specific tests, and make a similar PR there as well? What do you think?

Reply:
> That being said, should we take a look at enterprise specific tests, and make a similar PR there as well? What do you think?

That's a good point, we should. Did you merge master to enterprise-master?